### DISCUSSION TITLE
3rd CR Touch since upgrading firmware

### DISCUSSION THREAD

**Discussion by JamieLee2k** [mriscoc/Ender3V2S1#1097] [General]
So I am about to get my 3rd CR Touch and the reason is it keeps dropping half way through the print with the latest firmware, it ran fine with the old firmware and so I am pointing to this being the issue since nothing else has changed, is anyone else having this issue?

**Comment by mriscoc** [comment]
Check the wiring of your CRTouch.

**Reply by JamieLee2k** [reply]
The wiring was fine, when I changed the second one I kept the same wiring, this time I changed the probe and the wire so everything is new again but I don’t know what is causing it to drop half way through

**Reply by Quarkion** [reply]
Same thing for me, but on the left side of the print bed
